### ISSUE TITLE
a11y view should use all html file

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/chipper#1516]
It is currently card coded to use the _en file, which means translations cannot be tested in the a11y view in the built version. Lets update that and then re-generate files.

**Comment by jessegreenberg** [comment]
Done in https://github.com/phetsims/chipper/commit/cfb242f7daaf5e5a50e29d455c2b25148891dfc2 and then I regenerated files. Closingl
